### ISSUE TITLE
Body tag is doubled inside of the specs template via wrap.js

### ISSUE THREAD

**Issue by Teknologica** [sourcejs/Source#134]
While checking the source code to guide myself in my own SourceJS implementation I noticed that the template system was injecting the `body` tag into the `content` variable.

I doubled checked the live documentation to make sure it wasn't just my local and I found the same bug in the live version: taken from the source code of http://sourcejs.com/docs/spec/
```html
<div class="source_main source_col-main" role="main"><body><h1>Spec Page Documentation</h1>
```
After some digging through the code I found the source of the problem to be the `wrap.js` file, in particular the `process` function and the part handled via the `jsdom` call. 

I was able to fix it locally in `wrap.js` by re-assigning `specData` before the `templateJSON` is defined: 

```javascript
// make sure the body is not passed again once the head is removed
specData = specData.getElementsByTagName('body')[0];

// final data object for the template
var templateJSON = {
    content: specData.innerHTML,
// ...
```

I wasn't able to find a case where the `body` might be missing so I did not include a check like the one present for `headHook`.  

There's a mention in `wrap.js` that `JSDom` will be removed eventually, but this should fix it until then.

**Comment by operatino** [comment]
Hi,

Thanks for highlighting the issue and the solution provided. This comes from JSDom, that we will remove in upcoming 0.6.0. 

JSDom makes it's own DOM fixing, like browsers are, this is when additional body tag is injected. And we definitely don'y want to keep this magic, and leaving more space for bugs and leaks like that.

Could you please create a Pull Request with the fix you mentioned?

**Comment by operatino** [comment]
Merged into `0.5.3-dev`
